### DISCUSSION TITLE
Permalink renaming question

### DISCUSSION THREAD

**Discussion by Vetrenik** [squidfunk/mkdocs-material#5154] [Q&A] [answered]
Hello everybody!

Trying to make permalink work with non-latin letters, used https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/?h=perma#+toc.slugify this config, and it works just fine, but links, when copied, are coverted in this http://192.168.75.193:18080/en/about/architecture/#%E8%B7%A8%E6%B8%A0%E9%81%93

So i have a question - can i somehow make permalink respect this construction?
```markdown
# System-arch
## Cross-channel

## <a name="test" />跨渠道

[Testlink](#test)
```

Inside the page all works just fine, and cross-page links too
![image](https://user-images.githubusercontent.com/44965799/223062328-8495166c-df33-4fcf-bd6c-0f7e6fedaffe.png)

**Comment by squidfunk** [comment]
Yes. See our documentation on [`toc.slugify`](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#+toc.slugify).

**Reply by Vetrenik** [reply]
So, i need to write my own slug, not to use one of existing?

**Reply by squidfunk** [reply]
No. The documentation I linked mentions that you should use a Unicode-aware slug function that is included in `pymdowx`:

```yaml
markdown_extensions:
  - toc:
      slugify: !!python/object/apply:pymdownx.slugs.slugify
        kwds:
          case: lower
```

I'm not sure if you actually used that function because you did not mention it. You just said you used that setting.

**Reply by Vetrenik** [reply]
I actually used this function and all works correctly, but i'd like to have permalink give user value from ```<a name="test" />```block instead of giving smth like this http://192.168.75.193:18080/en/about/architecture/#%E8%B7%A8%E6%B8%A0%E9%81%93

**Reply by squidfunk** [reply]
Without knowing what you did exactly, we can't do anything beyond pointing you to our documentation. As with your other question, a [_minimal_ reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) is necessary for us to look into this.

**Comment by squidfunk** [comment, marked as answer]
Okay, so I think I initially misunderstood your request and I think you want this:

``` markdown
## 跨渠道 { #test }
```

Note that the [Attribute Lists extension](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#attribute-lists) must be enabled.

**Reply by squidfunk** [reply]
I'm marking this as an answer, as I'm confident this solves the problem. If not, please get back to me.
